package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nexmo-cli",
-  "version": "0.0.12",
+  "version": "0.0.13",
```

Comment:
I've reverted this in 61e337801d4e15851892ddd200975e527779ad18. Which then broke the build. Duh! So, fixed that in 4771d4b79f6990b6556c7ca6b9466448332223d5.